tests/virt_autotest/login_console.pm

```diff
@@ -15,15 +15,15 @@ use testapi;
 use Utils::Architectures;
 use Utils::Backends qw(use_ssh_serial_console is_remote_backend set_ssh_console_timeout);
 use ipmi_backend_utils;
-use virt_autotest::utils qw(is_xen_host check_port_state check_host_health);
+use virt_autotest::utils qw(is_xen_host is_kvm_host check_port_state check_host_health);
 use IPC::Run;
 
 sub set_ssh_console_timeout_before_use {
     reset_consoles;
     select_console('root-console');
     set_ssh_console_timeout('/etc/ssh/sshd_config', '28800');
     reset_consoles;
-    select_console 'sol', await_console => 1;
+    select_console 'sol', await_console => 0;
```

Comment:
What are the reasons for/benefit from disabling await_console?

Reply:
because I don't find there is any reason we have to wait for sol console on x86 at the moment. Sol console is not mandatory. @waynechen55 correct me if this is not correct.

Reply:
This await_console ensures that the console connection is setup successfully before it returns, and it actually is quick , only backend tries to build the connection with 'ipmitool xx sol'. Without this , it is possible that when `send_key 'ret'` in the next line is executed, the console is not ready yet.

Reply:
I just had a check in code. This function set_ssh_console_timeout_before_use is only used for arm case in login_console. Given that it has been running with await_console =>1 without problem, the sol needle should be there already. With the change in this PR to use await_console =>0, it will just loosen the condition for arm, no impact to x86 I think.

Anyway, it is not big deal. I suppose it is not the core of the this PR either. I do not insist on any option, just curious about the motivation/win for this change. I am willing to merge either of the final decision on this if there is benefit.

Reply:
From my verification run, I did not found a similar screen that can match previously 'sol' needle for arm. I did not find a suitable area suitable for 'sol' needle either on x86_64. Test run will assert 'linux-login' on sol which serves the same purpose, I think it would be better to do not use await_console with x86 otherwise it will introduce unexpected failures during test run, especially for login_console.pm.

Reply:
Agree.